lang/java/reef-tang/tang-test-jarA/src/main/java/org/apache/reef/tang/examples/A.java

```diff
@@ -19,5 +19,4 @@
 package org.apache.reef.tang.examples;
 
 public class A {
-  private int iAmAClass = 0;
```

Comment:
Yep. You're right. Those tests are not strongly related to this PR. As you mentioned above, we can revive `TestClassLoaders.java` after deleting `iAmAClass` fields.

I just want to point out that not only `A.iAmAClass` field but also the class `A` and `TestClassLoaders` is not used any more, too. I'm wondering if we need to remove all of them cleanly or to revive them. In anyway, sorry for distracting the focus of this PR.